src/apps/service/apps/de/limits.clj

```diff
@@ -0,0 +1,42 @@
+(ns apps.service.apps.de.limits
+  (:require [apps.clients.analyses :as analyses]
+            [apps.persistence.job-limits :as job-limits]))
+
+(def ^:private err-permission-needed "ERR_PERMISSION_NEEDED")
+(def ^:private err-forbidden "ERR_FORBIDDEN")
+(def ^:private err-limit-reached "ERR_LIMIT_REACHED")
+
+(defn- load-concurrent-vice-analysis-limit-check-results
+  "Loads the limit check for the number of concurrently running VICE analyses."
+  [user]
+  (let [limit-info (analyses/get-concurrent-job-limit (:shortUsername user))
```

Comment:
I know this is extremely minor, but for the sake of performance, it would be nice to make this HTTP call asynchronous by wrapping it in a `future` and then `deref` when it's used below, since the next line doesn't depend on this result and could be done in parallel.

Reply:
Good idea! I'll change that.